src/twisted/web/wsgi.py

```diff
@@ -455,6 +455,16 @@ def write(self, data):
         # Thing at some point in the future.
 
         def wsgiWrite(started):
+            if self._requestFinished:
+                # Self.request.notifyFinish() has triggered, and
+                # transport is potentially closed.
+                message = 'skipping write scheduled after transport closed'
+                self._log.error(message,
+                    system='wsgi',
+                    isError=True,
```

Comment:
maybe @wsanchez can help here.

are there any conventions for how to add extra data to an log message.

I was expecting that log.error() will take care of isError ... and self._log will take care of `system` so that we don't need to pass these arguments for each log message.

based on that reasoning system and isError are not adding any new information to the message

Reply:
I wasn't clear on the convention either. I copied a similar construct from the `_ErrorStream` class which may be used the the `wsgi` app for error logging.